platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -81,135 +81,113 @@ identity access, infrastructure operations, and app lifecycle.
+and success metrics, platform engineering teams in an enterprise must take ownership of delivering the required capabilities. 
+This typically leads into the evolution of the thinnest viable platform into an **internal platform** in cloud-native computing 
+paradigms. 
+
+Internal platforms are particularly relevant for cloud-native architectures because these
+separate supporting capabilities from application-specific logic more than
+previous paradigms. In cloud-like environments resources and capabilities are
+often managed independently and integrated with custom business components; such
+resources may include databases and object stores, Kubernetes clusters,
+observability collectors and dashboards and more. An internal platform provides
+these to enterprise teams in ways that make them easy to integrate in their
+applications and systems without having to have a deep innate knowledge about the underlying
+components, reducing cognitive load on internal stakeholders such as developers who can then focus 
+on the required work to deliver business outcomes. 
+
+For example, some of the use cases that can be met with an internal platform include: 
 
 1. Developers of products or services can automatically request a complete 
    development environment to support iterative research and development of new 
-   features. This includes spaces in relevant services such as task runners and 
-   artifact storage, membership in designated teams, and publication of connection 
+   features. This includes spaces in relevant services such as Kubernetes clusters, 
+   task runners and artifact storage, membership in designated teams, and publication of connection 
    info such as URLs and secrets.
-1. Developers of products or services can use scenario-specific code and 
-   configuration templates to rapidly bootstrap, develop and deliver new products 
-   and features
-1. Stakeholders in a product or service can observe system and user behavior in 
-   those products and services via instrumentation, dashboards and alerts
-
-A platform is bespoke to an organisation, supporting a unique set of users and 
-business needs. While application developement and delivery is often the first 
-capability a platform supports, it is important to focus on the individual needs of 
-the organization which often surfaces additional opportunities to optimise through 
-a platform offering. Some examples can include data operations, user management, and 
-incident management.
+1. Finops and other internal stakeholders get access to a dashboard for managing costs across business units with the ability to define chargeback rules without having to understand the underlying compute infrastructure 
+1. Operators of third-party products and services can automatically provision 
+   spaces and supporting services to deploy and use those third-party products and 
+   services
+
+These examples illuminate the definition of a platform according to Martin Fowler[[2]], 
```

Comment:
This was a quote from Bottcher on Fowler's blog. I have confirmed we cite correctly in the original, but I would be uncomfortable with only citing Martin here.

In addition, I fear that without the lead in of the quote, this is a bit of a throw away rather than the punchy statement that it was originally provided as. Was there a specific goal in mind by removing the industry quotes from the start of this section (both Atlassian and Bottcher/Fowler)?

Reply:
I had the same concern about the quote. The author of that article is Evan Bottcher, published on Martin Fowler's blog.